app/controllers/candidate_interface/candidate_interface_controller.rb

```diff
@@ -109,5 +110,13 @@ def append_info_to_payload(payload)
       payload.merge!({ candidate_id: current_candidate&.id })
       payload.merge!(query_params: request_query_params)
     end
+
+    def check_account_locked
+      if current_candidate&.account_locked?
+        sign_out(current_candidate)
+        flash[:error] = 'You\'re account has been locked'
```

Comment:
Do we need this if we're immediately redirecting to the `locked` page?

Reply:
Good point, don't think we need it, removed.